<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssessorWidget</class>
 <widget class="QWidget" name="AssessorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="assessorComboBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
	border: 0.05em solid #CA0B5E;
	border-radius: 0.25em;
	color: #CA0B5E;
	padding: 0.5em;
	font-size: 13pt;
	outline: none;
}
QWidget:focus {
	border: 0.1em solid #CA0B5E;
}
QWidget:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
QComboBox {
	background-color: transparent;
}
QComboBox QAbstractItemView:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QComboBox::item:selected
{
	background-color: rgba(202,11,94,0.8);
	color: white;
}
      </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>10</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
	border: 0.05em solid #CA0B5E;
	border-radius: 0.25em;
	color: #CA0B5E;
	padding: 0.5em;
	font-size: 13pt;
	outline: none;
}
QWidget:focus {
	border: 0.1em solid #CA0B5E;
}
QWidget:hover {
	background-color: rgba(202,11,94,0.8);
	color: white;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
        <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>150</width>
                <height>10</height>
            </size>
        </property>
        <property name="font">
            <font>
                <pointsize>13</pointsize>
            </font>
        </property>
        <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
            <bool>false</bool>
        </property>
        <property name="styleSheet">
            <string notr="true">QWidget {
                border: 0.05em solid #CA0B5E;
                border-radius: 0.25em;
                color: #CA0B5E;
                padding: 0.5em;
                font-size: 13pt;
                outline: none;
                }
                QWidget:focus {
                border: 0.1em solid #CA0B5E;
                }
                QWidget:hover {
                background-color: rgba(202,11,94,0.8);
                color: white;
                }
                QWidget:disabled {
                color: grey;
                border-color: grey;
                }
            </string>
        </property>
        <property name="text">
            <string>Cancel</string>
        </property>
        <property name="flat">
            <bool>true</bool>
        </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
